### DISCUSSION TITLE
Doorbell Pro

### DISCUSSION THREAD

**Discussion by ervwalter** [briis/unifiprotect#457] [General]
I expect to be installing my Doorbell Pro this weekend.  Any guesses on what to expect with the HA integration?  I know they sold out really quickly and are in short supply, so let me know if you need any help in testing these things.  I'm happy to make whatever data you need available.

**Comment by briis** [comment]
The built-in support in Protect is still a bit limited - No support for Fingerprint and NFC at the moment - so you can expect it to behave as the Non Pro version of the Doorbell right now. When HA 2022-02 comes out there will also be support for the Package Camera. Remaining support will be added when Ubiquiti adds the support in Protect.

**Reply by AngellusMortis** [reply]
The package camera is already supportee, but the entity is disabled by default. 2022.2.0 / 0.12.0-beta10 will improve it though so it is not disable by default. And remove streaming so there no buffering (2 FPS does not work with HLS streaming)
